src/configurations/destinations/webhook/schema.json

```diff
@@ -517,7 +517,7 @@
       },
       "webhookUrl": {
         "type": "string",
```

Comment:
make regex in consistent with ui-config.json one - this is incorrect one

Reply:
For textInput this part (`(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|`) is generated by default. If this is not present in `schema.json`, we are getting error in github actions